src/components/Editor/Tab.js

```diff
@@ -192,10 +192,7 @@ class Tab extends PureComponent<Props> {
         onContextMenu={e => this.onTabContextMenu(e, sourceId)}
         title={getFileURL(source)}
       >
-        <SourceIcon
-          source={source}
-          shouldHide={icon => ["file", "javascript"].includes(icon)}
-        />
```

Comment:
Could you restore the `shouldHide={icon => ["file", "javascript"].includes(icon)}` attribute?  We'd like to not show the JS or file icons because they'd be there 90%+ of the time.  Other than that, this awesome PR is good to go!

Reply:
Ah, I see.
sure no problem.